src/python/test/test_dxpy.py

```diff
@@ -1877,6 +1877,23 @@ def test_retry_after_without_header_set(self):
         self.assertTrue(50000 <= time_elapsed)
         self.assertTrue(time_elapsed <= 70000)
 
+    @unittest.skip("Example of a request that should be retried")
```

Comment:
I realized I can't modify the exception being thrown without resetting the call stack to that of the new throw site, and I didn't really want to go down that road (this relates to both the recent change to preserve the call stack properly, and also our other discussion about if there were a more useful way to chain exceptions).

In order to fix this test I've started catching the exception that's raised and checking that it passes `dxpy._is_retryable_exception` (but leaving the exception untouched in `DXHTTPRequest`).